content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Production environment"
+description: Create a production-quality Kubernetes cluster
+weight: 30
+no_list: true
+---
+<!-- overview -->
+
+A production-quality Kubernetes cluster requires planning and preparation.
+If your Kubernetes cluster is to run critical workloads, it must be configured to be resilient.
+
+<!-- body -->
+
+## Production issues
+
+Typically, a production environment Kubernetes has more requirements than a
+development or test environment Kubernetes. A production environment may require
+secure access by many users, consistent availability, and the resources to adapt
+to changing demands.
+
+As you decide where you want your production Kubernetes environment to live
+(on premises or in a cloud) and the amount of management you want to take
+on or hand to others, consider how your requirements for a Kubernetes cluster
+are influenced by the following issues:
+
+- *Availability*: A single-machine Kubernetes [learning environment](/docs/setup/#learning-environment)
```

Comment:
Ah, okay. Someone made the decision to divide Kubernetes setup into "Learning environment" and "Production environment" at some point. I think a better title than "Learning" would have been "Personal environment". Because those instructions would suit someone doing a personal learning, development, or test Kubernetes cluster. I'll change the sentence earlier where I talk about dev and test and lead with "personal learning environment". Hopefully, that will set up the learning environment tasks as more of a catch-all section for those kinds of non-production clusters.